### ISSUE TITLE
[Bug]: VARCHAR data at the maximum length is not allowed

### ISSUE THREAD

**Issue by kgeis** [milvus-io/pymilvus#1143]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

When I create a schema with a VARCHAR field and maximum length 4, then try to insert strings with length 4, I get the message:
> pymilvus.exceptions.ParamError:` invalid input, length of string exceeds max length. length: 4, max length: 4

### Expected Behavior

I should be able to insert strings of length 4 into a field with maximum length 4.

### Steps/Code To Reproduce behavior

```markdown
Create a collection with a VARCHAR field max length 4. Insert a string into it with length 4 in that field.
```

### Environment details

```markdown
- Hardware/Software conditions (OS, CPU, GPU, Memory): Ubuntu
- Method of installation (Docker, or from source): Python venv
- Milvus version (v0.3.1, or v0.4.0): Milvus 2.1.1, pymilvus 2.1.2
- Milvus configuration (Settings you made in `server_config.yaml`): default
```

**Comment by kgeis** [comment]
See [entity_helper.py](https://github.com/milvus-io/pymilvus/blob/master/pymilvus/client/entity_helper.py#L26):
`        if len(s) >= max_len:
`
Should probably be
`        if len(s) > max_len:
`

**Comment by izapolsk** [comment]
Hello team, it seems the fix of this issue hasn't been added to 2.2. Could you please include it in next release ?

**Comment by XuanYang-cn** [comment]
> Hello team, it seems the fix of this issue hasn't been added to 2.2. Could you please include it in next release ?

Sure, this would be released at pymilvus 2.2.3
